docs/en_US/NAS/Overview.md

```diff
@@ -93,6 +94,26 @@ cd ../darts
 python3 retrain.py --arc-checkpoint ../pdarts/checkpoints/epoch_2.json
 ```
 
+### ProxylessNAS
+
+The paper [ProxylessNAS: Direct Neural Architecture Search on Target Task and Hardware](https://arxiv.org/pdf/1812.00332.pdf) removes proxy, it directly learn the architectures for large-scale target tasks and target hardware platforms. It addresses high memory consumption issue of differentiable NAS and reduces the computational cost to the same level of regular training while still allowing a large candidate set.
```

Comment:
`It addresses high memory consumption` --> `It addresses the high memory consumption`

Reply:
I think remove `the` is better